python/ml4ir/applications/classification/tests/data/csv/validation/file_0.csv

```diff
@@ -0,0 +1,1001 @@
+"query_key","query_feature_0","group_feature_1","group_sequence_feature_2","entity"

```

Comment:
@Ullimague It will be useful to have more public facing names for these. My suggestion
```
query_key
query_text
domain_id
entity_id
user_context
```